File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        vector<int> arr(26);
        for(int i=0;i<s.size();i++){
            arr[s[i]-97]++;
        }
        for(int i=0;i<s.size();i++){
            if(arr[s[i]-97]==1){
                return i;
            }
        }
        return -1;
    }
};

File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
        int[] arr=new int[26];
        for(char c:s.toCharArray()){
            arr[c-97]++;
        }
        for(int i=0;i<s.length();i++){
            if(arr[s.charAt(i)-97]==1){
                return i;
            }
        }
        return -1;
    }
}